app/pathogen/sarscov2/layout.tsx

```diff
@@ -1,43 +1,48 @@
-import React from "react";
```

Comment:
Another gross diff :/

Only change is that I added
```

  if(!process.env.SARS_COV_2_TRACKER_ENABLED) {
    return notFound();
  }
```
on line 34. Everything else is the same.